docs/installing.rst

```diff
@@ -0,0 +1,84 @@
+Installation
+============
+
+Prerequisites
+-------------
+
+Docker
+~~~~~~
+Because of the many, many things that must be on your system for artman to
+do its job, artman utilizes `Docker`_ and ships a container which has
+appropriate dependencies available. You will, therefore, need Docker to be
+on your system.
+
+.. note::
+
+    It is also possible to run artman locally (use ``--local``) when invoking
+    it. This is generally only recommended for development of artman itself,
+    but is an option. If you do this, use our `Dockerfile`_ as a reference
+    to what dependencies you are likely to need.
+
+.. _`Docker`: https://docker.com/
+.. _`Dockerfile`: https://github.com/googleapis/artman/blob/master/Dockerfile
+
+googleapis
+~~~~~~~~~~
+Currently, artman is primarily used for building Google Cloud client libraries,
+and depends on a large, interdependent configuration structure. This is housed
+in the `googleapis`_ repository.
+
+In order to run artman, you will need to clone this repository, and generally
+you will want to be in this directory when invoking artman.
```

Comment:
s/will want to/should.

This is the current behavior though it is not by design. I need to fix an issue in artman.